scripts/.env

```diff
@@ -0,0 +1,2 @@
+MONGODB_URI="mongodb+srv://tap-admin:00J97H3T7JInU1TG@tap-cluster.qy3lv.mongodb.net/?retryWrites=true&w=majority&appName=TAP-Cluster"
+DB_NAME="tapDb"
```

Comment:
remove this, its exposing our database connection

Reply:
Removed